CHANGELOG.md

```diff
@@ -37,7 +37,6 @@ Usage:
 * Fix #1308: Add documentation for DeploymentConfigEnricher
 * Fix #1310: Add documentation + gradle integration test for ImageChangeTriggerEnricher
 * Fix #1325: `jkube.enricher.jkube-name.name` doesn't modify `.metadata.name` for generated manifests
-* Fix #1362: VolumePermissionEnricher : Replace iteration with bulk Collection.addAll call
```

Comment:
Why is this change required?

Reply:
Do i need to write there `Fix https://github.com/eclipse/jkube/issues/1374: Replaced the expected annotation with Assert.assertThrows in HealthCheckConfigTest.java for exceptions verification`?? Or it is not required?

Reply:
Ideally, we track changes via changelog. If you feel like doing it, you can add.